common/source/docs/common-camera-gimbal.rst

```diff
@@ -45,29 +44,29 @@ and to some extent this depends on the airframe. That said, using velcro
 and rubber bands (for extra security) is quick to set up and can give
 reasonable results.
 
-Connecting the servos to a Pixhawk
-==================================
+Connecting the servos to an Autopilot
+=====================================
 
-Connect the gimbal to Pixhawk's auxiliary output pins. Connect tilt
-(pitch) to aux out signal (s) pin 1, roll to aux out signal (s) pin 2,
-and ground to an aux out ground (-) pin.
+Connect the gimbal's roll, pitch(tilt), and/or yaw stabilization motor control inputs to the autopilot's output pins, as shown below for a Pixhawk.
 
 .. image:: ../../../images/pixhawk_to_gimbal_connection.jpg
     :target: ../_images/pixhawk_to_gimbal_connection.jpg
 
-Gimbal configuration through the mission planner
-================================================
+Gimbal configuration through Mission Planner
+============================================
 
-Under the configuration Menu, Hardware Options you will find a Camera
-Gimbal set-up screen. (see image below)
+Under the SETUP/Optional Hardware Menu, you will find a ``Camera
+Gimbal`` set-up screen. (see image below)
 
-For each servo/axis of your camera gimbal select the appropriate servo
-channel and ensure the **"Stabilise"** checkbox is checked.
+For each axis of your camera gimbal select the appropriate output
+channel that you connected above and ensure the appropriate **"Stabilise"** checkbox is checked.
 
 .. note::
 
    Modern brushless gimbals (like the :ref:`SToRM32 <common-storm32-gimbal>`) usually come with their own controllers that handle gimbal *stabilisation*. 
-   For those controllers the **Stabilise** checkbox must not be checked. 
+   For those controllers the **Stabilise** checkbox must not be checked. In this case the output channel will only control the desired attitude of the axis, the external controller will provide the stabilization of that attitude target.
```

Comment:
I think we should remove this entire "note".  I don't see why we should talk about a completely different type of gimbal on this servo gimbal page.